bats/gql/list-sessions.gql

```diff
@@ -0,0 +1,17 @@
+query userDetails {
+  me {
+    id
+    phone
+    mobileSessions {
+      id
+      expiresAt
+      issuedAt
+    }
+    delegations {
+      app
+      handledAt
+      remember
+      scope
+    }
```

Comment:
probably something to discuss at ACE or another meeting... why not just have a sessions query?

Reply:
you mean, as opposed to segmenting between delegation and sessions? we also have API keys btw.

I think the reason to not have them combined is because they have different properties.

for instance, delegated token can currently have smaller scope than a mobile token, which currently is always full access. delegated token are also attached to a particular application.